app/models/organization_table_info.rb

```diff
@@ -0,0 +1,165 @@
+class OrganizationTableInfo
+  attr_accessor :organizations_counts, :total_counts
+
+  def initialize(org_stats, total_counts)
+    @organizations_counts = org_stats.kind_of?(Array) ? org_stats : [ org_stats ]
+    @total_counts = total_counts
+  end
+
+  def self.top_stats(top: 0, organization: nil, weeks: 5)
+    # Retrieve the results count, payloads count and organization info for the top most consuming organizations
+    org_payloads_and_results_count = Organization
+      .select('
+        organizations.id, organizations.api_id, organizations.name, organizations.display_name,
+        SUM(test_payloads.results_count) AS test_results_count, COUNT(test_payloads.id) as test_payloads_count
```

Comment:
You have a mix of `as` and `AS` in this file's queries. I would suggest always using `AS`.